.github/workflows/swift-toolchain.yml

```diff
@@ -1668,15 +1709,18 @@ jobs:
                 -D SWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE=${{ github.workspace }}/SourceCache/swift-syntax `
                 -D SWIFT_PATH_TO_STRING_PROCESSING_SOURCE=${{ github.workspace }}/SourceCache/swift-experimental-string-processing
       - name: Build Swift Standard Library
+        if: ${{ !(matrix.os == 'Android' && inputs.build_os == 'Windows' && inputs.build_arch == 'arm64') }}
```

Comment:
Is this still necessary? IIUC, "Android built on Windows Arm64" is disabled with the matrix setup.

Reply:
This is testing if the build is "win/arm64" and the target is "android". I double-checked the matrix definition and I really believe this is never the case.

Reply:
Hmm? I see the matrix at lines 1460-1560 above for `stdlib` job, which doesn't seem to restrict Android/Win/ARM64 combination. Can you point me to the matrix definition you're referring to?

Reply:
Nevermind, my bad. I saw a couple of matrix inputs had been updated and I assumed this one had too.